### PR TITLE
Update Rails version check if Rails 7 or newer is not being used.

### PR DESCRIPTION
**Pull request by eclectic-coding**
@excid3 This removes Rails 6 installs #198 :
- Checked the method `rails_7_or_newer?` sooner so we can exit out before the install gets too far
- Included an error message and a note to delete the partial install directory

**One edge case**
We are not checking if the user overrides the default system Rails version: `rails _6.1.7_ new appname ...`.

### CODE REVIEW

**Comment by excid3** [comment]
> We are not checking if the user overrides the default system Rails version
What happens when they do that? Should still work right?

**Comment by eclectic-coding** [comment]
If their default system Rails version is `7`, the check would assume it is a Rails 7 install.

If the user overrides, as mentioned above, and the default system Rails version is `7`, it will fail as before.

If we would want to check for this edge case, we would need to figure out either how to read the version prefix in the cli args, or read from the already created `Gemfile`, which happens before the `template.rb` is accessed.

**Comment by eclectic-coding** [comment]
@excid3 I have addressed this edge case

Also, I removed the version check in the `add_users` method since we are now assuming Rails 7.

File: template.rb [hunk 1]
```diff
@@ -152,14 +162,14 @@ def add_multiple_authentication
 
   generate "model Service user:references provider uid access_token access_token_secret refresh_token expires_at:datetime auth:text"
 
-  template = """
+  template = "" "
```
Comment (excid3) [inline template.rb:165]:
```suggestion
  template = """
```

Comment (excid3) [reply template.rb:165]: Think this was changed on accident.

File: template.rb [hunk 2]
```diff
@@ -152,14 +162,14 @@ def add_multiple_authentication
...
   env_creds = Rails.application.credentials[Rails.env.to_sym] || {}
   %i{ facebook twitter github }.each do |provider|
     if options = env_creds[provider]
       config.omniauth provider, options[:app_id], options[:app_secret], options.fetch(:options, {})
     end
   end
-  """.strip
+  " "".strip
```
Comment (excid3) [inline template.rb:172]:
```suggestion
  """.strip
```

**Pull request merged**